library/std/src/sys/unix/process/process_fuchsia.rs

```diff
@@ -118,7 +118,8 @@ impl Command {
             FDIO_SPAWN_CLONE_JOB
                 | FDIO_SPAWN_CLONE_LDSVC
                 | FDIO_SPAWN_CLONE_NAMESPACE
-                | FDIO_SPAWN_CLONE_ENVIRON, // this is ignored when envp is non-null
+                | FDIO_SPAWN_CLONE_ENVIRON // this is ignored when envp is non-null
```

Comment:
Pre-existing, but is this comment perhaps wrong and should say "ignored when null"?

Reply:
I think the comment as-is is accurate. The flag is ignored if envp is set, meaning that an explicit environment is passed in. Otherwise, the FDIO_SPAWN_CLONE_ENVIRON flag clones the current process' environment.